src/AppInstallerRepositoryCore/Public/AppInstallerRepositorySearch.h

```diff
@@ -42,23 +42,43 @@ namespace AppInstaller::Repository
         Tag,
         PackageFamilyName,
         ProductCode,
+        NormalizedNameAndPublisher,
     };
 
     // A single match to be performed during a search.
     struct RequestMatch
     {
         MatchType Type;
         Utility::NormalizedString Value;
+        std::optional<Utility::NormalizedString> Second;

```

Comment:
>Second [](start = 49, length = 6)

nit: Second is not intuitive to me(as another match value) on first sight

Reply:
Additional?

Reply:
Changed to Additional